src/main/resources/assets/gtadditions/lang/en_us.lang

```diff
@@ -705,6 +706,18 @@ gregtech.machine.assembler.iv.name=Assembler IV
 gregtech.machine.autoclave.iv.name=Autoclave IV
 gregtech.machine.laser_engraver.iv.name=Laser Engraver IV
 
+gtadditions.machine.bundler.lv.name=Basic Bundler
+gtadditions.machine.bundler.mv.name=Advanced Bundler
+gtadditions.machine.bundler.hv.name=Advanced Bundler II
+gtadditions.machine.bundler.ev.name=Advanced Bundler III
+gtadditions.machine.bundler.iv.name=Advanced Bundler IV
+gtadditions.machine.bundler.luv.name=Elite Bundler
+gtadditions.machine.bundler.luv.tooltip=Lapotron 3000
```

Comment:
Lapinotron

Reply:
Right, I misremembered the name we wanted to use. Will fix that.